pkg/controller/workerroute/workerroute_controller.go

```diff
@@ -0,0 +1,291 @@
+/*
```

Comment:
Would be nice to maintain [the license](https://github.com/hellofresh/kubeflare/blob/master/hack/boilerplate.go.txt) used in the rest of the codebase.
I think `make generate` should use the expected license